.github/workflows/acceptance-tests.yml

```diff
@@ -236,6 +237,7 @@ jobs:
           MONGODB_ATLAS_ORG_ID: ${{ vars.MONGODB_ATLAS_ORG_ID_CLOUD_DEV }}
           MONGODB_ATLAS_BASE_URL: ${{ vars.MONGODB_ATLAS_BASE_URL }}
           MONGODB_ATLAS_ENABLE_BETA: ${{ vars.MONGODB_ATLAS_ENABLE_BETA }}
+          MONGODB_ATLAS_PROJECT_ID: ${{ vars.MONGODB_ATLAS_ACCTEST_STREAMS_PROJECT_ID_CLOUD_DEV }}
```

Comment:
can't it create the projects it needs?

Reply:
yeah, I think that's a good idea to also include the project resource here (create or reuse as the cluster), but I agree maybe better in a different PR

Reply:
created the PR: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1737. I will need to merge that PR here eventually because I see that my existing project created in dev was delete by clean up workflow, like the idea of not relying on existing projects altogether.